packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -142,6 +142,59 @@ class _AnimatedState extends State<AnimatedWidget> {
   Widget build(BuildContext context) => widget.build(context);
 }
 
+/// Signature for a convenience function for passing around a builder for a
+/// transiton's secondary animation.
```

Comment:
Nit: This sounds like too many "for"s to me, maybe just: Signature for a page transition's secondary animation builder?